Neuron/Models/Realm/WalletModel.swift

```diff
@@ -9,10 +9,29 @@
 import UIKit
 import RealmSwift
 
+enum WalleIconType: String {
+    case dog = "icon_wallet_dog"
+    case fish = "icon_wallet_fish"
+    case owl = "icon_wallet_owl"
+    case parrot = "icon_wallet_parrot"
+    case rat = "icon_wallet_rat"
+    case squirrel = "icon_wallet_squirrel"
+    case fox = "icon_wallet_fox"
+    case tiger = "icon_wallet_tiger"
```

Comment:
`icon_wallet` prefix should not be used in the raw value. Instead, add that prefix inside `var image`.